src/features.ts

```diff
@@ -18,6 +18,9 @@ const boolAsFeature = (currentState: boolean): FeatureState =>
 export const IsFeatureEnabled = (featureState: FeatureState): boolean =>
   featureState === FeatureState.Enabled;
 
+export const IsDynamicFeatureEnabled = (featureState: Dynamic<FeatureState>): boolean =>
+  featureState === ((): FeatureState => FeatureState.Enabled);
```

Comment:
I'm not sure if I'm missing something here, but I think this needs to be `featureState() === FeatureState.Enabled` to actually evaluate the setting at the time? And/or use `IsFeatureEnabled(features.REDmodAutoconvertArchives())` (I do sort of like the `IsDynamicFeatureEnabled` naming, it's more explicit than just a function call.)

The reasoning behind the types might be a little obtuse, I meant to leave them somewhat verbose to try to make sure we're always intentional about whether something is static or dynamic at the callsite, but it should be a matter of property access vs. function call.

Reply:
ah ya, I realized that later lol